src/Symfony/Bridge/Twig/AppVariable.php

```diff
@@ -89,6 +91,8 @@ public function getUser()
         if (null === $this->tokenStorage) {
             if (null === $this->container) {
                 throw new \RuntimeException('The "app.user" variable is not available.');
+            } elseif (!$this->container->has('security.context')) {
+                return;
```

Comment:
Actually, I know, but it will make the check in templates unusable if third-party bundles relies on it to activate templates sections when the `security` is configured.

`GlobalVariables` class, previously used for twig app globals in previous versions did this very same check and return `null` when the service doesn't exist. So, this is basically a BC break.